### PR TITLE
Replace IP-based URL with sosworkflows.com

### PR DESCRIPTION
**Pull request by joannfeng**
@BoPeng

### CODE REVIEW

File: doc/user_guide/concurrent_substep.html [hunk 1]
```diff
@@ -15494,7 +15482,7 @@ <h4>Substeps with statements after <code>sos_run</code> are not executed in para
 <script>
 function editme() {
     let filepath = window.location.pathname.slice(0, -5).replace('doc/', 'src/');
-    let new_url = `http://128.135.144.117:8000/hub/user-redirect/notebooks/${filepath}.ipynb`;
+    let new_url = `http://sosworkflows.com:8000/hub/user-redirect/notebooks/${filepath}.ipynb`;
```
Comment (joannfeng) [inline doc/user_guide/concurrent_substep.html:15485]: Is this what you meant? @BoPeng

Comment (BoPeng) [reply doc/user_guide/concurrent_substep.html:15485]: sosworkfliws.com serves on regular port, not on 8080.

Comment (BoPeng) [reply doc/user_guide/concurrent_substep.html:15485]: sosworkfliws.com serves on regular port, not on 8080.

Comment (BoPeng) [reply doc/user_guide/concurrent_substep.html:15485]: sosworkfliws.com serves on regular port, not on 8080.

Comment (joannfeng) [reply doc/user_guide/concurrent_substep.html:15485]: Does regular port mean the ":8000" should not be in the url?

Comment (BoPeng) [reply doc/user_guide/concurrent_substep.html:15485]: Yes, you should have tested that URL before using it.

**Comment by BoPeng** [comment]
We will see if it works in a few minutes (seconds).

**Pull request merged**